Formula/k/kubehound.rb

```diff
@@ -0,0 +1,37 @@
+class Kubehound < Formula
+  desc "Tool for building Kubernetes attack paths"
+  homepage "https://kubehound.io"
+  url "https://github.com/DataDog/KubeHound/archive/refs/tags/v1.3.3.tar.gz"
+  sha256 "5ee3ca8621d4611e682ed2325bb12b87d6f00c4f7717195d5f43856f5d9b7917"
+  license "Apache-2.0"
+
+  depends_on "go" => [:build, :test]
+  depends_on "docker" => :test
+
+  def install
+    goos = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOOS").chomp
+    goarch = Utils.safe_popen_read("#{Formula["go"].bin}/go", "env", "GOARCH").chomp
+
+    ldflags = %W[
+      -s -w
+      -X github.com/DataDog/KubeHound/pkg/config.BuildVersion=v#{version}
+      -X github.com/DataDog/KubeHound/pkg/config.BuildBranch=main
+      -X github.com/DataDog/KubeHound/pkg/config.BuildOs=#{goos}
+      -X github.com/DataDog/KubeHound/pkg/config.BuildArch=#{goarch}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "-o", "./bin/build/kubehound", "./cmd/kubehound/"
+    bin.install "bin/build/kubehound" => "kubehound"
```

Comment:
This defaults to installing in bin.
```suggestion
    system "go", "build", *std_go_args(ldflags:), "./cmd/kubehound/"
```

Reply:
Thanks, I didn't know that! I copied what I saw in some other formula that was using make, as I initially did